content/help/manual/create_account/index.md

```diff
@@ -3,106 +3,77 @@ title: Creating An Account
-your shared Family account or Company) as it will make it easier for others to
-identify you. This doesn't have to be your full or even your real name and can
-be changed at any time later on. It is also completely optional so you can just
-leave this empty for now if you want.
+You will receive a generated email with a 6-digit confirmation code which you can manual enter or copy and paste into the provided field.
 
-## Step 2: Confirm your email address
+{{< figure caption="Confirm your email address by entering the confirmation code." >}}
+{{< img src="3_signup_verify.png" >}}
+{{< /figure >}}
 
-As mentioned before, your email address will be used for things like two-factor
-authentication and will generally identify your account within Padloc, so we
-first need to make sure you actually own it. To do this, we will send you an
-email with 6-digit confirmation code. To proceed, simply enter this code in
-the provided field and click **Continue**.
+Click **submit**.
 
 {{< note >}}
-Note that email may take a few minutes before it shows up in your mailbox. You may
-also want to check your spam folder!
+**NOTE:**
+
+The email may take a few minutes before it shows up in your inbox. You may also want to check your spam folder!
 {{< /note >}}
 
-{{< figure caption="Confirm your email address by entering the confirmation code we'll send you." >}}
-{{< img src="signup_verify.png" >}}
-{{< /figure >}}
+## **Step 4: Create Your Master Password**
 
-## Step 3: Choose your master password
-
-This is the most important step of the signup process (don't worry, it is also
-the last) so you should pay close attention to what is happening here. Your
-**master password** is used to protect your account and all the information
-stored within it and although we use always use strong encryption to protect
-your data **this protection is only as strong as the master password you
-choose**. At the same time, **we will never store a copy of your password
-anywhere and will not be able to help you recover it** in case you forget it, so
-you want to make sure to use something that is both hard to guess and easy to
-remember. We know this is not an easy task, so the app will assist you by
-suggesting a strong password for you to use. This is a secure passphrase that
-is randomly generated from a list of the most common words in the English
-language (or whichever language you have set in your system preferences, as
-long as it is supported) using a variation of the [diceware
-method](https://ssd.eff.org/en/module/animated-overview-how-make-super-secure-password-using-dice).
-This passphrase may not look particularly secure, but we assure you it is
-likely much better than anything you would come up by yourself (no offence) and
-it certainly is a lot easier to remember than a string of random characters!
+This is the last password you **HAVE TO REMEMBER** and will be the most important password that you **NEED TO REMEMBER**.
```

Comment:
Is there a reason why you got rid of all this text? The manual is supposed to convey more information than just step-by-step instructions, and this information is quite important.

Reply:
Most of this text got moved to the "Read Me" text box at bottom - consolidated it there.

Reply:
And now have included them in these change updates.